pkgs/by-name/pa/parlay/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildGoModule rec {
+  pname = "parlay";
+  version = "0.5.1";
+
+  src = fetchFromGitHub {
+    owner = "snyk";
+    repo = "parlay";
+    rev = "v${version}";
+    hash = "sha256-9drACfJ8iA6qym7BQywJxzchQcFch4IW8/6q5dWb3KA=";
+  };
+
+  vendorHash = "sha256-Eo5MgdISiwbaJFg5XHAwe5x3D8GmgzswYmcUG4gvaQk=";
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Enriches SBOMs with data from third party services ";
```

Comment:
```suggestion
    description = "Enriches SBOMs with data from third party services";
```

Reply:
@kiike can you approve this change please?

Reply:
Hi! Thanks for the suggestion! I've committed the change and squashed it. You're attributed as co-author.